usage.md

```diff
@@ -8,45 +8,43 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
 
 ### Requirements
 
-The first three requirements derive from a placeholder install script. These will be addressed in #121.
+1. ESXi/vCenter - the target virtualization environment
+2. DHCP - the VCH currently requires there be DHCP on the external network (-extNet flag if not "VM Network")
+3. Bridge network - while install to vCenter environment, vic-machine does not create network dynamically, need to have separte bridge network (-bdgNet flag) specified, and should be different with external network
```

Comment:
Phrasing:
Bridge network - when installed in a vCenter environment `vic-machine` does not automatically create a bridge network. An existing vSwitch or Distributed Portgroup should be specified via the -bdgNet flag, and should not be the same as the external network.